src/Modules/Client.cpp

```diff
@@ -619,6 +637,16 @@ bool Client::Init() {
 
 	g_CalcViewModelLagHook.SetFunc(Client::CalcViewModelLag);
 
+	if (sar.game->Is(SourceGame_Portal2)) {
+#ifdef _WIN32
+		Client::AddShadowToReceiver = (decltype(Client::AddShadowToReceiver))Memory::Scan(client->Name(), "55 8B EC 51 53 56 57 0F B7 7D 08 69 FF ? ? ? ? 03 79 2C 51 89 4D FC 8B 0F 8B C4 89 08 8B 0D ? ? ? ? E8 ? ? ? ? 8B 75 0C 8B D8 3B DE 0F 84 ? ? ? ?");
```

Comment:
The last `? ? ? ?` can be removed because it's meaningless. I know there are other patterns that have it too but it should be done the correct way from now on.

Reply:
jesus christ why were my sigs so long lmao, anyway fixed